internal/resources/fabric/cloud_router/sweeper.go

```diff
@@ -64,7 +65,7 @@ func testSweepCloudRouters(region string) error {
 		if sweep.IsSweepableFabricTestResource(cloudRouter.GetName()) {
 			log.Printf("[DEBUG] Deleting Cloud Routers: %s", cloudRouter.GetName())
 			resp, err := fabric.CloudRoutersApi.DeleteCloudRouterByUuid(ctx, cloudRouter.GetUuid()).Execute()
-			if equinix_errors.IgnoreHttpResponseErrors(equinix_errors.HttpForbidden, equinix_errors.HttpNotFound)(resp, err) != nil {
```

Comment:
Are the function definitions of`HttpNotFound` and `HttpForbidden` being removed too?

Reply:
There's still a bit of packngo code remaining. I can see how it makes sense to wait before removing those functions entirely.

Reply:
No, those functions are still used in calls to the `IgnoreResponseErrors` function which was initially written for packngo responses and is still used by packngo-based code.